mhctools/cli/script.py

```diff
@@ -74,7 +83,10 @@ def main(args_list=None):
         input_dictionary = parse_fasta_dictionary(args.input_fasta_file)
         binding_predictions = predictor.predict_subsequences(input_dictionary)
     elif args.sequence:
-        binding_predictions = predictor.predict_peptides(args.sequence)
+        if args.extract_subsequences:
+            binding_predictions = predictor.predict_subsequences(args.sequence)
+        else:
+            binding_predictions = predictor.predict_peptides(args.sequence)
     elif args.input_peptides_file:
```

Comment:
I'd expect the `extract_subsequences` flag to be respect if the input is a file as well, so add it as an option here too? A bit weird for it to be inconsistent. Also, add a unit test for this?